framework/bin/gen_tests.pl

```diff
@@ -239,7 +239,7 @@ =head2 Logging
 $LOG->log_msg("Created test suite archive: $archive");
 
 # Acknowledge the tool author(s)
-system("cat $TESTGEN_BIN_DIR/$TOOL.credit");
+system("cat $TESTGEN_BIN_DIR/$TOOL.credit") if $DEBUG;
```

Comment:
I am fine with removing this from the output.

Can you delete the entire block and reference the tools in the README instead?

Reply:
OK, I have done that.  I think it's what you had in mind, but let me know if not.